### PR TITLE
Update Events example to use Chrono times

### PR DESCRIPTION
**Pull request by adbridge**
Also show how to use non_periodic periods and cancel() function.

### CODE REVIEW

**Comment by adbridge** [comment]
Ah this depends on : https://github.com/ARMmbed/mbed-os/pull/14087 being merged first

File: APIs_RTOS/Events_ex_1/main.cpp [hunk 1]
```diff
@@ -17,30 +19,69 @@ void handler(int count)
...
+//                    dispatched    ^       ^              ^
+//                                  ^       ^              ^
+//                               Event 5    ^              ^
+//                                          ^              ^
+//                                      Event 1,2,4        ^
+//                                       dispatched        ^
+//                                                         ^
+//                                                       Events 1,2,4
+//                                                        dispatched
+//
```
Comment (evedon) [inline APIs_RTOS/Events_ex_1/main.cpp:61]: That's great. Could you also show the expected output and perhaps print the time in `handler`?

File: APIs_RTOS/Events_ex_1/main.cpp [hunk 2]
```diff
@@ -17,30 +19,69 @@ void handler(int count)
...
+    // dispatch function. Note that the EventQueue library has yet to be
+    // converted to using Chrono times and thus times are still specified
+    // in integer millisecond units.
+    // 800 ms will allow the posted events to be dispatched multiple times
+    queue.dispatch(800);
 
     event_thread.join();
-}
+
+}
```
Comment (evedon) [inline APIs_RTOS/Events_ex_1/main.cpp:87]: Newline at end of file missing

**Review by evedon — COMMENTED** [review]
Great addition. I would add the expected output for greater clarity.

File: APIs_RTOS/Events_ex_1/main.cpp (hunk 1 above)
Comment (adbridge) [reply APIs_RTOS/Events_ex_1/main.cpp:61]: Yes that's a good idea

**Comment by adbridge** [comment]
@evedon PR updated, please re-review

**Review by evedon — APPROVED** [review]

File: APIs_RTOS/Events_ex_1/main.cpp (hunk 1 above)
Comment (0xc0170) [reply APIs_RTOS/Events_ex_1/main.cpp:62]: I like ascii art to show the sequence. I remember we did it in minar scheduler and it was quite clear what when should be triggered.

**Review by 0xc0170 — APPROVED** [review]

**Pull request merged**